### ISSUE TITLE
Possible login

### ISSUE THREAD

**Issue by faflfama** [shevabam/ezservermonitor-web#30]
Hi,
Besides the web interface working perfectly I dont want to make these informations aviable in public. Is there a possibility you can make this accessible only through some kind of login?

~Flam

**Comment by shevabam** [comment]
Hello,

This has not been planned at the beginning but may be considered in a future release.

However, you can put a .htaccess/.htpasswd quite simply.

**Comment by faflfama** [comment]
Yep, did that now, but I did it directly in the VHost. Thanks for the answer ;)

**Comment by gshafer** [comment]
Good instructions here on how to add password protection:

https://www.digitalocean.com/community/tutorials/how-to-set-up-password-authentication-with-apache-on-ubuntu-14-04
